.env.test

```diff
@@ -0,0 +1,15 @@
+ACCESS_TOKEN_COOKIE_NAME='edx-jwt-cookie-header-payload'
+BASE_URL='localhost:1995'
```

Comment:
Not that it matters but the base url is different here

Reply:
Ahhh. _Should_ this match with the `dev` env's port?
Would we need to worry about it  conflicting with the dev server?
And/or other test runs?

If there's a "best way" to do it, I can update here and back in the template too.

Reply:
Yeah, it's totally fine for it to be the same.  The test suites don't actually make any requests and don't run an actual server, this would only get used in URL generation.

Reply:
Or, rather, it should be the same for simplicity's sake. 🤪